browser/src/control/Control.JSDialog.js

```diff
@@ -602,7 +602,8 @@ L.Control.JSDialog = L.Control.extend({
 			var dialog = this.dialogs[autoFilterDialogId];
 
 			// Check if the current dialog has the isAutofilter property set to true
-			if (dialog.isAutofilter) {
+			// and if it is not an autoFillPopup
+			if (dialog.isAutofilter && (autoFilterDialogId !== 'autoFillPopup')) {
```

Comment:
Do we need to keep it open here? Maybe we don't need this change? Can you comment on this.